tests/org.palladiosimulator.dataflow.confidentiality.analysis.tests/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequencefinder/ActionSequenceFinderTest.java

```diff
@@ -34,6 +36,7 @@ public void testCount(StandalonePCMDataFlowConfidentialtyAnalysis analysis, int
         var allSequences = analysis.findAllSequences();
         assertEquals(expectedSequences, allSequences.size(),
                 String.format("Expected two dataflow sequences, but found %s sequences", allSequences.size()));
+        allSequences.stream().forEach(System.out::println);
```

Comment:
Is this code snippet in here by accident? E.g., because of local testing of the `toString()` method?

Reply:
Yes, this code snippet is in here to test the results of the changes. It was intended to be a logger.trace, so i will change this before this PR is merged